### PR TITLE
Update app-governance-get-started.md

### PR DESCRIPTION
**Pull request by WendyLiuC**

### CODE REVIEW

**Comment by PRMerger19** [comment]
@WendyLiuC : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: CloudAppSecurityDocs/app-governance-get-started.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ description: Get started with app governance capabilities to govern your apps.
 
 # Get started with app governance
 
+[Microsoft 365 licensing guidance for security & compliance.](https://aka.ms/ComplianceSD)
```
Comment (WendyLiuC) [inline CloudAppSecurityDocs/app-governance-get-started.md:10]: @dcurwin still learning how to edit PRs...but could you please change this to [Microsoft 365 security & compliance licensing guidance for app governance.](https://docs.microsoft.com/en-us/office365/servicedescriptions/microsoft-365-service-descriptions/microsoft-365-tenantlevel-services-licensing-guidance/microsoft-365-security-compliance-licensing-guidance#what-is-the-app-governance-add-on-feature-for-microsoft-defender-for-cloud-apps)

**Comment by ktoliver** [comment]
Hi @WendyLiuC - This pull request was opened in the public repo. PMs should work in the private repo, per the Microsoft Docs contributor guide. Would you make this content update in the private repo instead? Thank you! We'll close this PR.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

File: CloudAppSecurityDocs/app-governance-get-started.md (hunk 1 above)
Comment (dcurwin) [inline CloudAppSecurityDocs/app-governance-get-started.md:10]:
```suggestion
[Microsoft 365 security & compliance licensing guidance for app governance.](https://aka.ms/ComplianceSD)
```

**Review by dcurwin — APPROVED** [review]

**Comment by PRMerger19** [comment]
@dcurwin : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**